diff --git a/Makefile b/Makefile
index 340569a..236a3dc 100644
--- a/Makefile
+++ b/Makefile
@@ -99,37 +99,37 @@ $(TEST_EXECS_DWARF_ASM): %: %-dw.S
 	$(CC) $(TEST_SRC)/main.c $< -o $@
 
 odr-struct:
-	$(CXX) $(TEST_SRC)/odr.cc $(TEST_SRC)/odr-2.cc -I$(TEST_SRC) -o $@ -g \
+	$(CXX) $(TEST_SRC)/odr.cc $(TEST_SRC)/odr-2.cc -I$(TEST_SRC) -o $@ -gdwarf-4 \
 	  -DKIND=struct
 
 odr-class:
-	$(CXX) $(TEST_SRC)/odr.cc $(TEST_SRC)/odr-2.cc -I$(TEST_SRC) -o $@ -g \
+	$(CXX) $(TEST_SRC)/odr.cc $(TEST_SRC)/odr-2.cc -I$(TEST_SRC) -o $@ -gdwarf-4 \
 	  -DKIND=class
 
 odr-union:
-	$(CXX) $(TEST_SRC)/odr.cc $(TEST_SRC)/odr-2.cc -I$(TEST_SRC) -o $@ -g \
+	$(CXX) $(TEST_SRC)/odr.cc $(TEST_SRC)/odr-2.cc -I$(TEST_SRC) -o $@ -gdwarf-4 \
 	  -DKIND=union
 
 odr-struct-ns:
-	$(CXX) $(TEST_SRC)/odr.cc $(TEST_SRC)/odr-2.cc -I$(TEST_SRC) -o $@ -g \
+	$(CXX) $(TEST_SRC)/odr.cc $(TEST_SRC)/odr-2.cc -I$(TEST_SRC) -o $@ -gdwarf-4 \
 	  -DKIND=struct -DNAMESPACE=1
 
 odr-class-ns:
-	$(CXX) $(TEST_SRC)/odr.cc $(TEST_SRC)/odr-2.cc -I$(TEST_SRC) -o $@ -g \
+	$(CXX) $(TEST_SRC)/odr.cc $(TEST_SRC)/odr-2.cc -I$(TEST_SRC) -o $@ -gdwarf-4 \
 	  -DKIND=class -DNAMESPACE=1
 
 odr-union-ns:
-	$(CXX) $(TEST_SRC)/odr.cc $(TEST_SRC)/odr-2.cc -I$(TEST_SRC) -o $@ -g \
+	$(CXX) $(TEST_SRC)/odr.cc $(TEST_SRC)/odr-2.cc -I$(TEST_SRC) -o $@ -gdwarf-4 \
 	  -DKIND=union -DNAMESPACE=1
 
 odr-loc:
 	$(CXX) $(TEST_SRC)/odr-loc.cc $(TEST_SRC)/odr-loc-2.cc -I$(TEST_SRC) \
-	  -o $@ -g
+	  -o $@ -gdwarf-4
 
 
 def-decl:
 	$(CXX) $(TEST_SRC)/decl.c $(TEST_SRC)/def.c $(TEST_SRC)/def2.c -I$(TEST_SRC) \
-	  -o $@ -g
+	  -o $@ -gdwarf-4
 
 # On some systems we need to set and export DEJAGNU to suppress
 # WARNING: Couldn't find the global config file.
